projects/driver_controls/inc/center_console.h

```diff
@@ -6,16 +6,12 @@
 // Also dedicates another MCP23008 to LED output for state notification.
 //
 // Raises INPUT_EVENT_CENTER_CONSOLE_* events with empty data fields.
-// Assumes all inputs other than hazards are non-latching. The power button is required to be held
-// down for the specified period before a power event will be raised.
+// Assumes all inputs other than hazards are non-latching.
 #include <assert.h>
 #include <stdint.h>
 #include "gpio_expander.h"
 #include "soft_timer.h"
 
-// How long the power button must be held to count as a press
-#define CENTER_CONSOLE_POWER_HOLD_MS 2000
```

Comment:
Is it just an instant on now?

Reply:
Yeah, it was annoying to keep holding it. It's surprisingly hard to press the button.